std/node/_stream/duplex.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+export const errorOrDestroy = (...args: any[]) => {};
```

Comment:
What is this?  It isn't imported anywhere that I can detect.

Reply:
It is imported in the pipe method inside the readable stream. It's actually a placeholder for an upcoming function related to Duplex streams (PR added right after this one is approved).